.readme-partials/USING.md

```diff
@@ -41,7 +41,38 @@ To make use of the WP-CLI testing framework, you need to complete the following
 	```
 	This will make sure that the automated Behat system works across all platforms. This is needed on Windows.
 
-5. Update your composer dependencies and regenerate your autoloader and binary folders:
+5. Optionally add a `phpcs.xml.dist` file to the package root to enable code style and best practice checks using PHP_CodeSniffer.
+
+    Example of a minimal custom ruleset based on the defaults set in the WP-CLI testing framework:
+    ```xml
+    <?xml version="1.0"?>
+    <ruleset name="WP-CLI-PROJECT-NAME">
+	<description>Custom ruleset for WP-CLI PROJECT NAME</description>
```

Comment:
Indentation looks off here:
```suggestion
    <description>Custom ruleset for WP-CLI PROJECT NAME</description>
```

Reply:
Good catch. Turns out it was completely mixed throughout the `USING` file, so I'll add a (separate) commit normalizing the indentation for all text and code samples in that file.